src/LayerStyler/StyleManager/index.js

```diff
@@ -166,7 +167,14 @@ class StyleManager extends Component {
       return Object.values(dedup).map(group => group.reverse())
     }
     const layerTitles = layers.map(getTitleForLayer)
-    const layerSelected = activeIdx !== null
+    const layerSelected = activeIdx !== null // eslint-disable-line
+    let tooltipMsg = ''
+
+    if (!layerSelected) {
+      tooltipMsg = 'Select a Geoserver Layer to filter attributes'
+    } else if (!layers[activeIdx].isGeoserverLayer) {
+      tooltipMsg = 'VectorLayers dont have filter capabilities, only Geoserver Layers do'
+    }
```

Comment:
move to translations

Unsupported layer type for filtering Layer attributes (Supported types include GeoserverLayer)